packages/components/src/navigation/styles/navigation-styles.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
+const horizontalPadding = '30px';
+
+export const Root = styled.div`
+	width: 272px;
+	padding: 35px 0px;
```

Comment:
I'm seeing 32px 

![Screen Shot 2020-08-17 at 12 05 34 PM](https://user-images.githubusercontent.com/1922453/90347070-0efc2f80-e082-11ea-9cff-e9836cda5c4e.png)

Reply:
Removed this padding.  Other GB components don't seem to include any level of padding like we're doing here.